Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/SDK/Python/general_utilities.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
```

Comment:
This whole document needs a deprecation notice in the header! (we are going to deprecate SDK in future iteration.)